rust/lance-arrow/src/lib.rs

```diff
@@ -366,6 +366,9 @@ pub trait RecordBatchExt {
     /// If the named column does not exist, it returns a copy of this [`RecordBatch`].
     fn drop_column(&self, name: &str) -> Result<RecordBatch>;
 
+    /// Replace the column specified with the name and return the new [`RecordBatch`].
```

Comment:
```suggestion
    /// Replace a column (specified by name) and return the new [`RecordBatch`].
```
Minor nit but my brain parsed this originally as `replace <the column> with <the name>`  and I though initially it was some kind of rename operation.